engine/execution/ingestion/engine.go

```diff
@@ -581,6 +586,8 @@ func (e *Engine) executeBlock(ctx context.Context, executableBlock *entity.Execu
 
 	startedAt := time.Now()
 
+	e.stopControl.executingBlockHeight(executableBlock.Block.Header.Height)
```

Comment:
I got your idea, good point. We only need to track executing height, rather than processable height.